lib/gui/components/update-notifier/styles/_update-notifier.scss

```diff
@@ -20,3 +20,7 @@
   margin-bottom: 0;
 }
 
+.modal-update-notifier .modal-body a {
```

Comment:
You mean in terms of styling? It's just a static blue; there's no visual feedback here. It should be easy to add after we've standardized colours like you suggested below though!